include/cudaq/Optimizer/Transforms/Passes.h

```diff
@@ -36,10 +36,12 @@ createApplyOpSpecializationPass(bool computeActionOpt);
 std::unique_ptr<mlir::Pass> createDelayMeasurementsPass();
 std::unique_ptr<mlir::Pass> createExpandMeasurementsPass();
 std::unique_ptr<mlir::Pass> createLambdaLiftingPass();
+std::unique_ptr<mlir::Pass> createLiftArrayAllocPass();
```

Comment:
```suggestion
```
Use `createLiftArrayAlloc()`. It's already declared.

Reply:
Done, thanks!